### ISSUE TITLE
Python has stoped working

### ISSUE THREAD

**Issue by rezaizanloo** [Kaixhin/grokking-pytorch#1]
This line of my code 
correct += (predicted[0] == labels[0]).sum().item()
cause Python has stoped working.

# 6. Calculate Accuracy
correct = 0
total = 0
with torch.no_grad():
    for data in testloader:
          images, labels = data
          outputs = net(images)
          _,predicted = torch.max(outputs.data,1)
          total +=labels.size(0)
        

> When my code reach to this line, the widows kill the python kernel.
![image](https://user-images.githubusercontent.com/45669156/49604650-b404d080-f9a3-11e8-99e9-7116867bb474.png)

print('Accuracy of the Network is :  %d' %(100*correct/total))

**Comment by Kaixhin** [comment]
I've checked my code, which runs fine, as well as the code which you've displayed here which is different, but this also runs fine for me. This shouldn't have anything to do with the code I've provided, so please post your issue on the [PyTorch forums](https://discuss.pytorch.org/) for help on resolving this.
